templates/github/.github/workflows/update_ci.yml.j2

```diff
@@ -34,12 +34,12 @@ jobs:
         with:
           committer: {{  'pulpbot <pulp-infra@redhat.com>' }}
           author: {{  'pulpbot <pulp-infra@redhat.com>' }}
-          branch: ${GITHUB_REF#refs/heads/}-update-ci-files
-          base: ${GITHUB_REF#refs/heads/}
-          title: 'Update CI for ${GITHUB_REF#refs/heads/}'
+          branch: {{ plugin_default_branch }}-update-ci-files
+          base: {{ plugin_default_branch }}
+          title: 'Update CI files'
```

Comment:
Was this supposed to update the files on release branches too?

Reply:
Eventually I am hoping to handle that. But for now, it just handles `plugin_default_branch`.

Reply:
Ug yes, good catch. I suppose I need to find a way to handle that. The problem is that `${GITHUB_REF#refs/heads/}` doesn't resolve here. I suppose I need to ditch this `peter-evans/create-pull-request@v3` and write a script to create a PR.

Reply:
I don't think you need to ditch it, it works here:
https://github.com/pulp/plugin_template/blob/master/templates/github/.github/workflows/create-branch.yml.j2#L52-L74
I believe you need to create the branch first, and 
```
git checkout -b $BASE_BRANCH-update-ci-files
echo "BASE_BRANCH=some logic before" >> $GITHUB_ENV
```